src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -79,6 +79,7 @@ public enum ByteCode :byte
         POPSCOPE       = 0x5b,
         STOREEXIST     = 0x5c,
         PUSHDELEGATE   = 0x5d,
+        BRANCHTRUE     = 0x5e,
```

Comment:
I chose to add this to the end instead of next to BRANCHFALSE because if I moved all the enum values down one, it would break KSM.